modules/core/misc/java/test/CoreTest.java

```diff
@@ -962,9 +962,9 @@ public void testMahalanobis() {
 
         assertEquals(0.0, d);
 
-        d = Core.Mahalanobis(line1, line2, covar);
-
-        assertTrue(d > 0.0);
+        // Bug: https://github.com/opencv/opencv/issues/24348
+        // d = Core.Mahalanobis(line1, line2, covar);
+        // assertTrue(d > 0.0);
```

Comment:
It makes sense to disable this check for ARM only.

Reply:
Thanks a lot for the suggestion! I'm debugging the issue right now and hopefully propose a solution in a couple of days.